crane-cloud/frontend
# Add graph data

# Description
Add billing graph data

## Type of change


## Trello Ticket ID
https://trello.com/c/USen0tZ6

## How Can This Been Tested?

Go to the billing page and check out the graph, it may not show because the metrics are very insignificant but you will see values on the y axis to show it is working.

## Checklist:


## Screenshots
![Screenshot from 2022-09-19 19-20-04](https://user-images.githubusercontent.com/32802973/191079528-6f79c2c4-f08b-4f78-a66b-185b807e4ba7.png)
![Screenshot from 2022-09-19 19-19-49](https://user-images.githubusercontent.com/32802973/191079536-cf2f1243-cbfd-4b97-b124-f1dc2277acb6.png)
![Screenshot from 2022-09-19 19-19-40](https://user-images.githubusercontent.com/32802973/191079537-b49b569a-de18-483a-95e9-2a2a75d30de1.png)
![Screenshot from 2022-09-19 19-19-26](https://user-images.githubusercontent.com/32802973/191079540-b16c1825-8ace-4cdc-a7d5-c0d445fc9d36.png)
